README.md

```diff
@@ -1 +1,14 @@
 # flask-blog-poc
+Flask Microblogging PlatForm.
+
+For running application follow steps given below:
+	
+	1. create virtualenv and activate it.
+	2. pip install -r requirements.txt.
+	3. create database in postgresql.
+	4. export DATABASE_URL="postgresql://username:password@localhost:5432/database-name"
```

Comment:
Please categorised the settings files configuration for Development, Production & Testing. 
**For e.g**. : Create settings structure like : 
settings/local/DevelopmentConfig ==> for local development.
settings/local/ProductionConfig  ==> For production use
settings/local/TestConfig   ===> For testing